template-for-repository/proofs/Makefile.common

```diff
@@ -125,9 +125,9 @@ endif
 #     CHECK_FLAG_POINTER_CHECK =
```

Comment:
I noticed that I made made a typo in my PR which introduced this comment. This line should be:

```
#     CHECK_FLAG_POINTER_CHECK :=
```

This would define the `CHECK_FLAG_POINTER_CHECK` immediately (which would be later detected by `?=`).

Sorry!

Reply:
I just tested this on my machine. `=` also works :) So ignore this ^ comment.